content/applications/finance/accounting/get_started/shared_accounts.rst

```diff
@@ -0,0 +1,137 @@
+===============
+Shared Accounts
+===============
+
+In Odoo, accounts can be shared between multiple companies. This feature is particularly useful for
+businesses that operate under a **multi-company structure** and might benefit from a
+**unified chart of accounts**.
+
+Two ways are available to set up said feature: the :ref:`basic setup <setting_up_shared_accounts>`
+or using the :ref:`merge tool <merge_tool>`.
+
+The merging tool allows existing multi-company environments to use the new shared accounts feature.
```

Comment:
```suggestion
The merge tool allows existing multi-company environments to use the new shared accounts feature.
```

You call it the "merge tool" above, so let's keep things homogeneous. If we prefer "merging tool", it's fine with me as well :)